apps/root/src/pages/home/components/portfolio/index.tsx

```diff
@@ -64,6 +67,26 @@ const StyledNoWallet = styled(ForegroundPaper).attrs({ variant: 'outlined' })`
   `}
 `;
 
+const StyledTableEnd = styled(TableCell).attrs({ size: 'small' })`
+  ${({ theme: { spacing } }) => `
+    padding: ${spacing(1)} 0px !important;
+  `}
+  height: 1px;
+`;
+
+const StyledLink = styled(Link)`
```

Comment:
want to move this to a component? seems like we'll be using it in most tables, like "AnimatedChevron"

Reply:
Yeah! For some reason, we are getting 'rendered more hooks than expected' in virtualized rows. Work around this issue by avoiding the use of hooks